module/VuFind/src/VuFind/Controller/ContentController.php

```diff
@@ -55,28 +55,45 @@ public function contentAction()
             ->getLocale();
         $defaultLanguage = $this->getConfig()->Site->language;
 
+        $types = [
```

Comment:
Would it make sense to move this to a class property so it's easier to extend with additional types?